setup.py

```diff
@@ -34,7 +34,7 @@
     description="Command line interface for Mbed OS.",
     keywords="Arm Mbed OS MbedOS cli command line tools",
     include_package_data=True,
-    install_requires=["python-dotenv", "Click==7.0", "mbed-devices", "mbed_tools_lib"],
+    install_requires=["python-dotenv", "Click==7.0", "mbed-devices", "pdoc3"],
```

Comment:
We do need `mbed-tools-lib` for the `log_exception` function.

Reply:
https://github.com/ARMmbed/mbed-tools/pull/7/commits/470c3143173b2c6332bff8fa0b4f20c73ced164c